src/core/Chip/BaseChip/BaseChip.baseStyles.tsx

```diff
@@ -21,15 +21,15 @@ export const baseChipBaseStyles = (theme: SuomifiTheme) => css`
     padding: ${theme.spacing.insetXxs} ${theme.spacing.insetL};
     color: ${theme.colors.whiteBase};
     background: ${theme.colors.highlightBase};
-    max-height: 28px;
+    max-height: 100%;
```

Comment:
There is no need for Max-height: 100%. This is used only if parent has max-height value and child needs to be same height.

Reply:
You can remove whole max-height

Reply:
We have no control over the context where chips are used. Would it make sense to limit the max size to fit the container and trigger ellipsis if texts cannot fit?